### DISCUSSION TITLE
Hacktoberfest 2019

### DISCUSSION THREAD

**Discussion by tcelestino** [frontendbr/forum#1551] [Discussão]
Para quem não conhece, o Hackotberfest é um "evento" patrocionado pela a DigitalOcean para quem quiser contribuir com diversos _pull request_ em diversos projetos de diferentes linguagens no Github. Além de poder contribuir para esses projetos, ao atingir uma cota minima de _pull request_ aprovados, você é premiado com uma camiseta. Sem sorteio! E enviada para qualquer lugar do mundo!

Esse ano, o [Dev.to](https://dev.to) se juntou a DigitalOcean e está sendo patrocionadora do hackathon

Para participar, basta se inscrever no link: https://hacktoberfest.digitalocean.com/

Também acontece em alguns lugares do mundo eventos que reunem o pessoal para resolver PRs em conjunto. Caso tenha interesse em participar, basta acessar https://hacktoberfest.digitalocean.com/events e verificar se ocorrerá em uma alguma cidade próxima a você.

A pergunta que deixo para quem participou ou não: esse tipo de iniciativa te faz querer contribuir com a comunidade _open source_? E por quê?

:crab:

**Comment by mstuttgart** [comment]
Sim. Eu tenho o costume de contribuir com projetos _opensouce_ que normalmente utilizo, embora não contribua com a frequência que gostaria. 

O _hacktoberfest_ é uma boa oportunidade pra descobrir projetos novos pra contribuir e também chamar novos contribuidores para nossos projetos pessoais. Eu mesmo tenho projetos em que nos eventos anteriores, outras pessoais enviaram contribuições muito relevantes (documentação, código, testes e etc) e também houve projetos em que não contribuiria normalmente (seja pela linguagem, por ser algo que não uso e etc) e que reservei um tempo pra contribuir com algo.

**Comment by lucasferreiralimax** [comment]
Eu acho uma boa para encontrar novos projetos e aprender um pouco sobre novas iniciativas, todo ano eu divulgo meu projeto open-source mas nunca recebeu PR hahaha.

Lembrando que apenas os primeiros 50.000 participantes serão legíveis para a t-shirt.

**Comment by wilsonneto-swe** [comment]
Opa, participando também. eu contribui com certa frequência seja com código ou com doações para aqueles projetos que me ajudam no dia a dia, seja sobre trampo ou apps que facilitam a vida.

Participo pra unir o últil ao agradável e ganhar os colecionáveis hehe

Abraços!

**Comment by eduardosoareszup** [comment]
@tcelestino Eu não entendi direito, para ser elegível de ganhar a camisa tem que criar um PR com a tag `hacktoberfest` e ele ser aprovado? É isso?

**Comment by rafaelsevla** [comment]
@eduardosoareszup nao.. vc so tem q fazer login no hacktober com seu github e fazer qualquer pull request (menos os pr inutil como colocar comentario ou apagar linha em branco) 

nao precisa ser aprovado

**Comment by rafaelsevla** [comment]
bem q a galera poderia postar os projetos opensource aqui pra gente contribuir ne? heheeh

**Comment by eliseumds** [comment]
@rafaelsevla eles recomendam alguns: https://hacktoberfest.digitalocean.com/#projects, mas você pode enviar PR pra qualquer projeto público que esteja no GitHub.

**Comment by rafaelsevla** [comment]
@eliseumds sim sim eu vi kk queria contribuir com projetos brazuka tb

**Comment by eliseumds** [comment]
@rafaelsevla oh, pode crer, seria legal ter uma lista dessa.

**Comment by eduardosoareszup** [comment]
O ruim é que a maioria dos projetos que a galera recomenda já estão estáveis, acho que seria mais legal contribuir com algum que esteja mais no início ou que desse para participar da criação de bastante coisa. Sei lá, daria um sentimento de contribuição maior, pelo menos para mim.

**Comment by rafaelsevla** [comment]
@eliseumds @eduardosoareszup oq vcs acham de abrir uma issue pra galera postar o link dos repositorios?

**Comment by eduardosoareszup** [comment]
@rafaelsevla tem alguns repositórios que a galera criou para isso 

https://github.com/frontendbr/open-source/tree/master/projetos
https://github.com/camilatigre/listamaravilhosaopensource

O ruim que alguns estão parados. E sei lá, tá difícil achar um que eu me identifico :(

**Comment by rafaelsevla** [comment]
opa nao tinha visto kkk 
depois vou dar uma olhada

cara eu tenho um aqui q eu fiz pra guardar todas as mini chatisses q fazemos diariamente tipo regex mascaras etc pra ficar mais facil de achar... se quiser contribuir nele:
https://github.com/rafaelsevla/universal-utilities

**Comment by adeonir** [comment]
pessoal, observem para não contribuir com repositórios que pedem para incluir o nome e um perfil ou coisa do tipo

esses estão sendo considerado spam e não vão contabilizar

**Comment by eduardosoareszup** [comment]
@rafaelsevla eu tenho um aqui em que eu estava tentando converter `json` em `html`, ainda não sei se o conceito seria interessante, acho que só tento um MVP para poder validar se seria mais produtivo ou não. Porém eu empaquei na hora de fechar as tags html, se você tiver alguma ideia de como fazer isso pode acabar me ajudando 😃 

[Repositório](https://github.com/eduardosoareszup/html-super/tree/development), por enquanto esotu desenvolvendo na branch de dev.

**Comment by eduardosoareszup** [comment]
@tcelestino cara respondendo a sua pergunta. Acabou que me fez começar a contribuir, comecei a procurar mais projeto que possam estar precisando de ajuda e ando pensando em como ajuda-los. Nunca tinha contribuído, então fui pesquisar um pouco em como fazer e até que não é o bicho de 7 cabeças que eu achei que seria. Estou gostando bastante.

Aaaah é, obrigado por abrir essa Issue, acho que senão fosse por ela eu não teria começado essa jornada. Espero que ela seja longa kk.

**Comment by felipefialho** [comment]
@eduardosoareszup só essa sua resposta já valeu a abertura dessa issue, sensacional @tcelestino 😻

**Comment by ninetails** [comment]
Só pra adicionar, a [Globo](https://opensource.globo.com/hacktoberfest/) também participa e pra camiseta deles (não do evento) é 2 PR, sendo 1 aprovado. Mas não vi muita coisa relacionada a frontend.

**Comment by brunooomelo** [comment]
Infelizmente, eu hypei demais com o **Hacktoberfest** e agora perdi a vontade de fazer algum PR e por falta de tempo tambem.

mas acho fantastico quem consegue contribuir.
